micromamba/tests/test_create.py

```diff
@@ -713,8 +730,6 @@ def test_root_prefix_precedence(
         if set_in_condarc:
             f.write(f"envs_dirs: [{str(condarc_envs_dirs)}]")
 
-    # TODO: Remove this call to `os.makedirs` once
-    # https://github.com/mamba-org/mamba/issues/3790 is fixed
     for envs_folder in (condarc_envs_dirs, conda_envs_dirs, cli_provided_root, mamba_root_prefix):
         os.makedirs(envs_folder, exist_ok=True)
 
```

Comment:
Same comment. 
```suggestion
```

Can you check that all of those directories are present on the filesystem at the end of the test?

Reply:
Good catch - whoops, I did mean to take those lines out! 

We already check for the existence of the directories here: [L750-L757](https://github.com/mamba-org/mamba/pull/3796/files#diff-63e099f2fcceb7d321e6823ad7e744dfc92cdb4dea2b5507bc9d090ab9f09e53L750-L757)